package.json

```diff
@@ -120,6 +120,7 @@
     "@babel/preset-typescript": "^7.12.7",
     "@babel/runtime": "^7.12.5",
     "@compodoc/compodoc": "^1.1.14",
+    "@discoveryjs/json-ext": "^0.5.3",
```

Comment:
why is this needed?

Reply:
I don't know, I just thought we were supposed to put it at the root as well.

Reply:
i don't know what--if anything--needs to be at the root. we should probably clean it out at some point. in the meantime, let's not make it worse unless it's needed.